apps/condo/domains/common/components/ShowMoreFieldsButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react'
+import { Space, Typography } from 'antd'
+import styled from '@emotion/styled'
+
+import { useIntl } from '@core/next/intl'
+
+import { ChevronIcon } from '@condo/domains/common/components/icons/ChevronIcon'
+import { Button } from '@condo/domains/common/components/Button'
+
+import { colors } from '../constants/style'
+
+type ChevronIconWrapperProps = {
+    direction: 'down' | 'up',
+}
+
+const ChevronIconWrapper = styled.div<ChevronIconWrapperProps>`
+    transform: rotate(${props => props.direction === 'down' ? 0 : 180}deg);
+    display: flex;
+`
+
+const SHOW_MORE_BUTTON_STYLE = { padding: 0 }
```

Comment:
It has grey background on hover, that does not disappears after click and mouse out.
Maybe to remove grey background, because without padding it look bad?

Reply:
it's better to inline such things